package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "joi-to-swagger",
-  "version": "1.2.0",
+  "version": "1.3.0",
```

Comment:
Please remove this change. The version number will be incremented by `npm version` when I tag the release.